.gitignore

```diff
@@ -1,10 +1,18 @@
 config.ini
+config_*
 reference_transcriptions.csv
+reference_transcriptions_*
 stt_transcriptions.csv
 wer_details.csv
 wer_summary.json
+<<<<<<< Updated upstream
 wer_word_accuracy.csv
+=======
```

Comment:
Will need to fix this merge conflict.

Reply:
I think that got added when I did a "stash".  I removed the bad line.